packaging/installer/kickstart.sh

```diff
@@ -868,6 +871,7 @@ try_package_install() {
       else
         pm_cmd="yum"
       fi
+      SYSVERSION=$(echo "$SYSVERSION" | cut -d'.' -f1)
```

Comment:
This should probably go up in the `get_system_info()` function instead of here.

Reply:
we cannot remove the stuff after the dot for all operating systems, as Opensuse has 15.2, 15.3 versions and need to stay like that..

Reply:
I don’t mean do it for all platforms, just have the special case for it up in `get_system_info()` instead of in the code that attempts to install a native package.

`try_package_install()` is not guaranteed to be called,  and the `SYSVERSION` variable is one of the things that gets reported by telemetry, so there’s a risk of handling this here resulting in inconsistencies in our anonymous statistics. By mangling the version in `get_system_info()` instead, we avoid that possibility (and also ensure that every other part of the script that may care can count on the value being consistent no matter what execution path is taken).

Reply:
agreed. I will make this change now